pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -47,7 +47,7 @@ import (
 	netutil "k8s.io/apimachinery/pkg/util/net"
 	certutil "k8s.io/client-go/util/cert"
 	cloudprovider "k8s.io/cloud-provider"
-	v1helper "k8s.io/kubernetes/pkg/apis/core/v1/helper"
+	cloudutil "k8s.io/cloud-provider/util"
```

Comment:
@cheftako now is your chance to suggest a better package name here :P

Reply:
Maybe `k8s.io/cloud-provider/node` ?

Reply:
@andrewsykim @cheftako any conclusion on this? `k8s.io/cloud-provider/node` seems ok :)

Reply:
We're having a formal discussion on this here https://github.com/kubernetes/community/pull/2628 but my gut tells me `k8s.io/cloud-provider/node` is going to be fine so if you're okay to do the work and maybe change it later then go ahead :)

Reply:
@islinwb sorry for late reply! I think `k8s.io/cloud-provider/node` is 👍

Reply:
Let's go with `k8s.io/cloud-provider/node` for now. Can always update later :)